duet/dashboards/firefox_mr1_launch.dashboard.lookml

```diff
@@ -0,0 +1,2932 @@
+- dashboard: firefox_mr1_launch
+  title: Firefox MR1 Launch
+  layout: newspaper
+  preferred_viewer: dashboards-next
+  load_configuration: wait
+  refresh: 1 day
+  elements:
+  - title: Uplift Desktop
+    name: Uplift Desktop
+    model: DUET
```

Comment:
Tested on a fork.
```
    model: duet
```
also needs to be changed for all other occurrences

Reply:
updated and confirmed